### PR TITLE
DTCT-211 payment update integration guide

### PR DESCRIPTION
**Pull request by hstrychalski**

### CODE REVIEW

File: pages/03.fe-development/09.subscriptions/docs.md [hunk 1]
```diff
@@ -180,6 +191,70 @@ Status modifications use a [POST /subscription/status](https://docs.centra.com/s
...
+
+```json
+{
+  "status": "ok",
+  "paymentHtml": "<div id=\"klarna-checkout-container\" (...) </div>",
+  "contractIds": [1, 2, 3]
+}
+```
+
+After receiving the response, the update should be reflected on the subscription contract response. Newly created subscription payment depending on the status can/cannot/cannot be used yet for placing recurring orders.
```
Comment (piotr-symonowicz) [inline pages/03.fe-development/09.subscriptions/docs.md:256]: Can we re-phrase the second sentence? Not sure what it means

File: pages/03.fe-development/09.subscriptions/docs.md [hunk 2]
```diff
@@ -262,3 +337,69 @@ Status modifications use a [POST /subscription/status](https://docs.centra.com/s
...
+```json
+{
+  "token": "aeea9561d2653df51017ccdac3cdac38",
+  "status": "ok",
+  "paymentHtml": "<div id=\"klarna-checkout-container\" (...) </div>",
+  "contractIds": [1, 2, 3]
+}
+```
+
+After receiving the response, the update should be reflected on the subscription contract response. Newly created subscription payment depending on the status can/cannot/cannot be used yet for placing recurring orders.
```
Comment (piotr-symonowicz) [inline pages/03.fe-development/09.subscriptions/docs.md:404]: Same as above - can/cannot/cannot?

**Review by piotr-symonowicz — COMMENTED** [review]
Looks good with tiny changes, please re-phrase those two sentences

File: pages/03.fe-development/09.subscriptions/docs.md [hunk 3]
```diff
@@ -262,3 +337,69 @@ Status modifications use a [POST /subscription/status](https://docs.centra.com/s
...
+
+```json
+{
+  "contract": 1
+}
+```
+
+### Subscription payment update initialization
+
+To initialize subscription payment update use Checkout API endpoint [POST /api/checkout/customers/stored/payment](https://docs.centra.com/swagger-ui/?api=CheckoutAPI#/6.%20customer%20handling/post_customer_stored_payment_methods).
```
Comment (sirtopp) [inline pages/03.fe-development/09.subscriptions/docs.md:354]: `customer/stored/payment`

File: pages/03.fe-development/09.subscriptions/docs.md (hunk 1 above)
Comment (hstrychalski) [reply pages/03.fe-development/09.subscriptions/docs.md]: rephrased it, it was supposed to be:

- can
- cannot
- cannot be used yet

but yeah that was not clear enough :D

Comment (hstrychalski) [reply pages/03.fe-development/09.subscriptions/docs.md]: or clear at all

File: pages/03.fe-development/09.subscriptions/docs.md (hunk 3 above)
Comment (hstrychalski) [reply pages/03.fe-development/09.subscriptions/docs.md]: done

File: pages/03.fe-development/09.subscriptions/docs.md [hunk 4]
```diff
@@ -262,3 +337,69 @@ Status modifications use a [POST /subscription/status](https://docs.centra.com/s
...
+{
+  "token": "aeea9561d2653df51017ccdac3cdac38",
+  "action": "form",
+  "formHtml": "<div id=\"klarna-checkout-container\" (...)"
+}
+```
+
+### Subscription payment update result
+
+To finalize subscription payment update use Checkout API endpoint [POST /api/shop/customer/stored/payment-result](https://docs.centra.com/swagger-ui/?api=CheckoutAPI#/6.%20customer%20handling/post_customer_stored_payment_methods).
```
Comment (sirtopp) [inline pages/03.fe-development/09.subscriptions/docs.md:379]: checkout

**Pull request merged**